CONTRIBUTING.md

```diff
@@ -86,10 +86,34 @@ component of the Thanos distributed system. We recommend:
 `make help` will print most of available commands with relevant details.
 * Spin up a prebuilt dev environment using Gitpod.io
 [![Gitpod Ready-to-Code](https://img.shields.io/badge/Gitpod-ready--to--code-blue?logo=gitpod)](https://gitpod.io/#https://github.com/thanos-io/thanos)
+* In case you want to develop the project locally, install **Golang** in your machine. Here is a nice [gist](https://gist.github.com/nikhita/432436d570b89cab172dcf2894465753) for this purpose.
+
+### Installing Project locally in your machine
+
+* Change your directory to the gopath for installing Thanos there -
+  * `cd $GOPATH/src/github.com/`
+* If you don’t have `$GOPATH` set, then add them -
+  * `export GOPATH=$HOME/go; export PATH=$PATH:$GOROOT/bin:$GOPATH/bin`
+  * Make sure you add the exports to the `/.bashrc` file so that those environment variables are persisted when you run them again.
```

Comment:
```suggestion
  * Consider adding the environment variables to your host machine (e.g `/.bashrc` or [`.envrc`](https://direnv.net/)) file so that those environment variables are persisted across sessions.
```

Reply:
Done